werkomgeving/gegeven/index.md

```diff
@@ -63,21 +63,21 @@ Ook kunnen er gegevens opgenomen worden in het gegevensobject over andere object
 
 ## Een gegevensobjecttype
 
-Een gegevensobjecttype is *een typering van gelijksoortige gegevensobjecten*. Met een gegevensobjecttype beschrijven we hoe een afzonderlijk gegevensobject er uit ziet, als groepering van gegevens. Van een gegevensobjecttype wil je de volgende zaken weten:
+Een gegevensobjecttype is *een typering van gelijksoortige gegevensobjecten*. Met een gegevensobjecttype beschrijven we hoe een afzonderlijk gegevensobject er uit ziet, als groepering van gegevens. Van een gegevensobjecttype wil je de volgende zaken weten: 
 
 1. Welke gegevenstypen zijn onderdeel van dit gegevensobjecttype?
-2. Welke gegevenstypen zijn beoogd voor de identificatie van het object waarover de gegevens gaat? (dwz: welke gegevenstypen vormen de sleutel)
```

Comment:
Zou het ook kunnen dat een geheel gegevensobject identificerend is voor een object? Zo ja, neem dan ook dit op: ‘_Is het gegevensobject identificerend voor het object waar het gegevensobject betrekking op heeft?_’.